packages/amplify_core/lib/src/types/query/query_field_operators.dart

```diff
@@ -220,7 +220,7 @@ class ContainsQueryOperator extends QueryFieldOperatorSingleValue<String> {
     } else if (other is List<String>) {
       return other.contains(value);
     } else {
-      throw const AmplifyException(
+      throw const DataStoreException(
```

Comment:
Query fields can be used outside of Datastore in model helpers for API category. Is there another exception type that could be used here?

Reply:
Good catch! Created a new `ModelQueryException` type which extends `AmplifyException`